### PR TITLE
Book: Update published topics

### PR DESCRIPTION
**Pull request by randomvariable**
Had some missing topics on the website, this should fix.

### CODE REVIEW

File: docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md [hunk 1]
```diff
@@ -42,9 +43,9 @@ clusterawsadm alpha bootstrap create-stack \
 These will be added to the control plane and node roles respectively when they are created.
 
 **Note:** If you used `clusterawsadm` 0.5.4 or earlier to create IAM objects for the Cluster API
-Provider for AWS, using `clusterawsadm` 0.5.5 or later will, by default, remove the bootstrap 
+Provider for AWS, using `clusterawsadm` 0.5.5 or later will, by default, remove the bootstrap
 user and group. Anything using those credentials to authenticate will start experiencing
-authentication failures. If you rely on the bootstrap user and group credentials, specify 
+authentication failures. If you rely on the bootstrap user and group credentials, specify
```
Comment (benmoss) [inline docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md:48]: This is my favorite part

Comment (randomvariable) [reply docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md:48]: Only if you move from the alpha commands to the GA commands. The alpha command behaviour was preserved...

Comment (benmoss) [reply docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md:48]: I meant the whitespace removal 😸

Comment (randomvariable) [reply docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md]: In any case, I've clarified the language.

**Review by vincepri — APPROVED** [review]
/approve

**Pull request merged**